packages/login/network/BitcoinEngine.js

```diff
@@ -53,7 +53,7 @@ export class BitcoinEngine {
     const { inputs, outputs, fee } = coinselect(utxos.map((output) => ({
       txId: output.txid,
       vout: output.vout,
-      value: output.satoshis,
+      value: Number.parseFloat(output.satoshis),
```

Comment:
If there is INTEGER value, why `Number.parseFloat(output.satoshis)` not the Number(output.satoshis) ?
If there is a Decimal value in the string - why float, not BigNumber?